packages/excalidraw/element/textElement.ts

```diff
@@ -347,6 +353,10 @@ const getLineWidth = (
   font: FontString,
   forceAdvanceWidth?: true,
 ) => {
+  if (text === "") {
+    return 0;
+  }
```

Comment:
I don't remember if we discussed the reason behind this at some point, but I'm not sure if it's still necessary?

Either way, this is breaking measurement in a way that new text elements don't display a caret at all until you add a character.

It may also break something else that I'm not aware of. (Ditto for the same in `measureText()`).

Reply:
This is needed such that if the match is at the beginning of a line, the offset to the start won't be a nonzero value. But we could also revert the changes in `textElement.ts` and place the check in `SearchMenu` instead.